griptape/drivers/prompt/cohere_prompt_driver.py

```diff
@@ -128,6 +128,7 @@ def _base_params(self, prompt_stack: PromptStack) -> dict:
                 else {}
             ),
             **({"preamble": preamble} if preamble else {}),
+            **self.extra_params,
```

Comment:
question for all, should `extra_params` be able to overwrite all other parameters?

Reply:
While it could allow users to shoot themselves in the foot, I don't think we should block behavior.